includes/gateway/class-omise-payment-billpayment-tesco.php

```diff
@@ -0,0 +1,148 @@
+<?php
+defined( 'ABSPATH' ) or die( 'No direct script access allowed.' );
+
+function register_omise_billpayment_tesco() {
+	require_once dirname( __FILE__ ) . '/class-omise-payment.php';
+
+	if ( ! class_exists( 'WC_Payment_Gateway' ) || class_exists( 'Omise_Payment_Billpayment_Tesco' ) ) {
+		return;
+	}
+
+	/**
+	 * @since 3.7
+	 */
+	class Omise_Payment_Billpayment_Tesco extends Omise_Payment {
+		public function __construct() {
+			parent::__construct();
+
+			$this->id                 = 'omise_billpayment_tesco';
+			$this->has_fields         = false;
+			$this->method_title       = __( 'Omise Bill Payment: Tesco', 'omise' );
+			$this->method_description = wp_kses(
+				__( 'Accept payments through <strong>Tesco Bill Payment</strong> via Omise payment gateway.', 'omise' ),
+				array( 'strong' => array() )
+			);
+
+			$this->init_form_fields();
+			$this->init_settings();
+
+			$this->title       = $this->get_option( 'title' );
+			$this->description = $this->get_option( 'description' );
+
+			add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, array( $this, 'process_admin_options' ) );
+			add_action( 'woocommerce_thankyou_' . $this->id, array( $this, 'display_barcode' ) );
+		}
+
+		/**
+		 * @see WC_Settings_API::init_form_fields()
+		 * @see woocommerce/includes/abstracts/abstract-wc-settings-api.php
+		 */
+		public function init_form_fields() {
+			$this->form_fields = array(
+				'enabled' => array(
+					'title'   => __( 'Enable/Disable', 'omise' ),
+					'type'    => 'checkbox',
+					'label'   => __( 'Enable Omise Tesco Bill Payment', 'omise' ),
+					'default' => 'no'
+				),
+
+				'title' => array(
+					'title'       => __( 'Title', 'omise' ),
+					'type'        => 'text',
+					'description' => __( 'This controls the title which the user sees during checkout.', 'omise' ),
```

Comment:
`This controls the title the user sees during checkout.`

Reply:
@jonrandy  👍 you've mentioned that for Installment once, I forgot 😂  
Thanks!